Shop-G/ShopGo
# build onboarding screen

## jira
[TS-5](https://shop-go.atlassian.net/browse/TS-5)

Description:
* build the welcome onboarding screen for shopGo.
* autolayout has been done in both screens.

How to test it?
* run the app in simulator and the welcome screen will appear followed by the main screen. press the home icon on the top of a simulator which will take you to the main page of simulator. once you are on home screen of a simulator, swipe up the screen to close the app. after closing it, reopen the app  one more time in order to see the changes that were implemented in the onboarding screen. when the app opens, it will skip the welcome screen and the main screen of logIn/createAccount will appear. 
* once you see the main view controller, delete the app from simulator and run it again through xcode. the welcome screen will appear again followed by the main screen.
